include/geometry/point2ll.h

```diff
@@ -10,16 +10,13 @@ Integer points are used to avoid floating point rounding errors, and because Cli
 */
 #define INLINE static inline
 
-// Include Clipper to get the ClipperLib::IntPoint definition, which we reuse as Point definition.
-#include <cmath>
-#include <functional> // for hash function object
-#include <iostream> // auto-serialization / auto-toString()
+//#include <functional> // for hash function object
+//#include <iostream> // auto-serialization / auto-toString()
 #include <limits>
 #include <polyclipping/clipper.hpp>
-#include <stdint.h>
+//#include <stdint.h>
 
```

Comment:
Then you should not add it in this header, but in the cpp file that actually needs it. `Point2LL` currently has a lot of "inline" function but they should be in a cpp file, and this one should include `numbers`. Again, the purpose is to avoid exponential header inclusion.

Reply:
My point was not that the way I fixed it was great, just to point out that there was a problem 😛

Reply:
Indeed, but I can't really fix that because I don't build on Windows, so I don't know in which file to add the missing include.